Client/Coordinators/CoordinatorFlagManager.swift

```diff
@@ -26,4 +26,10 @@ struct CoordinatorFlagManager {
         && LegacyFeatureFlagsManager.shared.isFeatureEnabled(.libraryCoordinatorRefactor,
                                                              checking: .buildOnly)
     }
+    
+    static var isEtpCoordinatorEnabled: Bool {
+        return CoordinatorFlagManager.isCoordinatorEnabled
```

Comment:
Since we'll rely as well later on the settings coordinator, we can check that flag is on instead of the `CoordinatorFlagManager.isCoordinatorEnabled`